### DISCUSSION TITLE
Install an extension on Mirth Connect v4.4. Is it possible?

### DISCUSSION THREAD

**Discussion by arulferiandevops** [nextgenhealthcare/connect#6143] [Q&A] [answered]
Hello everyone,
I am a new Mirth Connect user, and I have a question. Can I install an extension (for example, the LDAP extension)? Where can I download the file? Is it only available for Premium?
Thank you.

Mirth Connect v.4.4.5

**Comment by pacmano1** [comment, marked as answer]
The LDAP plug-in is a premium extension and requires a license.

Licenses for extensions are grouped in license levels. The best resource to talk to is Nextgen sales.

Once you have a license, you'll be given access to download the plug-ins.

**Comment by jonbartels** [comment]
There are some open-source Mirth extension developers as well. 

- https://github.com/ChristopherSchultz (Has an LDAP plugin)
- https://github.com/tonygermano
- https://github.com/kpalang
- https://github.com/jonbartels

**Reply by pacmano1** [reply]
Thanks @jonbartels, I often forget @ChristopherSchultz 's LDAP and MFA work at https://github.com/ChristopherSchultz/mirth-plugins

**Reply by pacmano1** [reply]
Note that for Nextgen premium extensions, even if you could find them, they require a license key to work.

**Comment by arulferiandevops** [comment]
Thanks @pacmano1 and @jonbartels
